common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -366,10 +366,14 @@ export interface ISignalClient {
 
 // @public (undocumented)
 export interface ISignalMessage {
+    // (undocumented)
+    clientConnectionNumber?: number;
```

Comment:
I know the existing properties aren't documented, but could you add TSDocs for the new properties?